### PR TITLE
Add sanctions for d0ku-era Assault Rifles

### PR DESCRIPTION
**Pull request by carlst99**
This sanctions the d0ku-era assault rifles as `infantry`, bringing them in-line with other burst assault rifles.
It also classes the `Equinox VE2` as `none`, in-line with other underbarrel-equipped assault rifles.

It would be good to have a discussion about the d0ku-era weapons, as they have quirks in general. Particularly of concern for the weapons in this PR is the incendiary launcher on the `MG-A1 Arbalest`, which falls very much into cheese territory. My argument for sanctioning the gun as 'infantry' is that the launcher is one-time use, rather ineffective and anecdotally used by few players (much like other d0ku-weapon attachments that fall into the quirk category). Otherwise, the weapons are functionally similar to other burst rifles.

### CODE REVIEW

**Comment by fredkilbourn** [comment]
I don't know enough about these weapons, would love to hear more feedback from others.\

**Comment by fredkilbourn** [comment]
One observation I had: since the Yumi is sanctioned and it has an underbarrel, maybe we can bring back the other ones?  Or, we should ban the Yumi.  Either way to bring consistency.  Thoughts?

**Comment by cooltrain7** [comment]
I've never used any of them so I can't comment on _not_ having them.

**Comment by brakenium** [comment]
The Lacerta VS D0ku AR has a higher skill ceiling than non-burst AR's IMO. Once you can consistently hit the head it becomes way more effective. Hitting body shots is usable but you'd be better of with a different (auto) weapon. While I do have the other AR's unlocked, I haven't used them enough to say much. But I can say from my limited experience is that the while the NC variant seems about as good as VS (with bullet drop). The TR variant's 6 round burst is too long, making it less effective. I can't say anything about attachments.
TLDR; Lacerta is a mostly skill based weapon because of head shots

**Review by cooltrain7 — APPROVED** [review]
I have no issues with these being tagged for inf, we can always review it again in the chance anyone disagrees

**Review by fredkilbourn — APPROVED** [review]
I thinking adding the Equinox VE2 'none' with the 'Gauss Rifle S' and the 'T1S Cycler' is the most consistent choice for now, but later we should examine the disparity of these being banned and the Yumi not.

PSA-02 Diamondback makes sense as it's just an NS-11A reskin.

The 3 Doku ARs seem fine, I don't know enough but I think the only new weapon that has a repuation for being OP is the Horizon so when we review Carbines we will need to have a bigger discussion about that.
